src/hooks.tsx

```diff
@@ -137,6 +137,9 @@ export const useWindowSize = (): {
     );
 
     React.useEffect(() => {
+        if (typeof window === 'undefined') {
+            return;
+        }
```

Comment:
in which case window will be undefined? If you're thinking in SSR, useEffects don't run in SSR

Reply:
oh, excellent. thanks